TestProjects/NotificationSamples/Assets/Scripts/Helpers/PlatformSelector.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PlatformSelector : MonoBehaviour
+{
+    void Awake()
+    {
+        #if PLATFORM_ANDROID
```

Comment:
I think you should actually use a swtich with  Application.platform instead of defines, that why you'll get compile errors if there are any in advance

Reply:
Done in latest commit